sdk-base/src/main/java/com/mapbox/maps/plugin/delegates/MapCameraManagerDelegate.kt

```diff
@@ -19,6 +19,8 @@ interface MapCameraManagerDelegate {
   /**
    * Convenience method that returns the camera options object for given arguments
    *
+   * This API isn't supported by Globe projection.
```

Comment:
What's an alternative though?

Reply:
there isn't :)

Reply:
we are looking to re-evaluate such APIs and potentially deprecating them

Reply:
Maybe explicitly suggest to use Mercator then? Also - link to the Globe in javadoc, with `[Globe]` / `[Mercator]`